### DISCUSSION TITLE
How to "restore" oc_jobs after they were deleted

### DISCUSSION THREAD

**Discussion by rhatguy** [nextcloud/recognize#959] [Q&A] [answered]
I've removed all of the background jobs entries from my oc_jobs table at some point and now it seems recognize doesn't run automatically anymore.  How can I re-create the background scanning jobs in the oc_jobs table?  I tried re-installing recognize, but that doesn't seem to do anything.

**Comment by marcelklehr** [comment, marked as answer]
You can run `occ recognize:recrawl`. This will recreate the necessary cron jobs.
